testapi.pm

```diff
@@ -903,7 +903,7 @@ sub script_sudo {
 
 =head2 script_output
 
-  script_output($script [, $wait, type_command => 1])
+  script_output($script [, $wait, type_command => 1, die_on_failure => 1])
```

Comment:
I know I'm late in the game but we usually show case how to pass non-default parameters. And if I understand it correctly, 1 is default. You want to allow the opposite by adding this paramter.

But having a die_on_failure => 0 sounds double negative. So give this a positive meaning - and make 0 the default. From what I get: proceed_on_failure => 1

Reply:
the problem that `proceed_on_failure` which really ease understanding on function input param level , complicate things in actual place of usage - this is why I decided to go with `die_on_failure`. But let's try to follow you suggestion ...